src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +10,18 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private static final UserService userService = new UserService();
```

Comment:
```suggestion
    private final UserService userService = new UserService();
```
don't overuse static, let's just keep it final, it will become clear why when Spring Boot section starts